src/frontend/ocamlmerlin/ocamlmerlin.c

```diff
@@ -521,30 +522,84 @@ static void compute_merlinpath(char merlin_path[PATHSZ], const char *argv0, stru
 }
 
 #ifdef _WIN32
+
+/* May return NULL */
+char * retrieve_user_sid_string()
+{
+  char * usidstr = NULL;
+
+  void * process_token = NULL;
+  if ( ! OpenProcessToken( GetCurrentProcess(), TOKEN_QUERY, &process_token ) )
+    return usidstr;
```

Comment:
Personally, I would explicitly `NULL` here, rather than subtly via a variable
```suggestion
    return NULL;
```